duckduckgo/iOS
# UX/UI improvements

Task/Issue URL: https://app.asana.com/0/856498667320406/1116850244629757

**Description**:
- Progress bar color updated to cornflower blue.
- Updated Primary Text color (table cell text color) and Search Bar/Toolbar icon colors in order to have better contrast. Also made some tweaks around Navigation Bar items color.
- Increased search bar font size.
- Updated Table Cells selection color.
- We are now using custom Disclosure Indicator so we can customize its color.
- Updated Browse Next/Prev button images.

**Steps to test this PR**:
For both Light theme and Dark theme:

Search bar tweaks:
1. Enable centered search with favorites on Home Screen. 
2. Make sure transition to top bar is smooth and text size/colors are matching.

Note: search field background color for 'centered' and 'in-top-bar' modes are slightly different - this is intended.

Other:
- Make sure tab image (Lottie one) matches other buttons' colors.
- Table cells should now be readable when highlighted.




---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
